### DISCUSSION TITLE
inlineEditColor

### DISCUSSION THREAD

**Discussion by danappsnocode** [cortexrd/Knack-Toolkit-Library#110] [General]
I have made a few changes to my Knack look and feel. I notice the background color for inline edit columns has change. Must be something I have done - but I don't know what. 

What controls the background color for inlineEdit Columns? 

Dan B

**Comment by cortexrd** [comment]
Hi Dan,

The "inline editable" color is driven by the app's header color by default.  But you can change that behavior to your liking in many ways.
You can derive it from another theme's element color, as well as a hard-coded value.

In the KTL_Default.js file, search for this:
**ktl.systemColors.setCfg**

The variable **inlineEditBkgColor** is the one you're interested in.

You must copy this code block to your customized KTL code:

```
ktl.systemColors.getSystemColors()
    .then(sc => {
        ktl.systemColors.setCfg({
            inlineEditBkgColor: '', //Could also be some other presets in sysColors like: sc.paleLowSatClrTransparent
            inlineEditFontWeight: '', //Can be 'bold' or a numeric value like 600.
            tableRowHoverBkgColor: '', //Or a named color like 'mistyrose' or a hex RGBA value '#a0454b75'
        })
    })
    .catch((err) => { ktl.log.clog('red', 'App getSystemColors error: ' + err); })
```

See this Wiki section on how to customize your KTL features:  [https://github.com/cortexrd/Knack-Toolkit-Library/wiki/Customizing-Features](url)

Once you've done this, you can decide to use a custom color like this:
(just the line above)

`inlineEditBkgColor: 'cadetblue',`

or

`inlineEditBkgColor: '#5f9ea0', //same as cadetblue but in hex format.`

It's also possible to create your own theme-based derived colors, with lower saturation, brighter, etc.

Hope this helps,
Norm

**Comment by danappsnocode** [comment]
Great!

Thanks *again* Norm.

Dan

Dan Boultinghouse
APPS*nocode*.*com*

**Comment by danappsnocode** [comment]
NOT AN ISSUE.  Did not want ot post on Github. just fyi

ZOOM says only for views.

working great for Edit forms popup or not.   Love it.  Elderly volunteers
at food pantry 😀


Dan Boultinghouse
APPS*nocode*.*com*
